sklearn/naive_bayes.py

```diff
@@ -384,17 +421,11 @@ def _partial_fit(self, X, y, classes=None, _refit=False,
             # Put epsilon back in each time
             self.sigma_[:, :] -= epsilon
 
+        _check_y_classes(y, self.classes_)
```

Comment:
This is alright. Ideally we would not perform unique on y multiple times, which I think is currently what this helper would do if used in the batch (fit) case. (Not sure I stated that clearly...)